sdk/eventhub/event-hubs/tests.yml

```diff
@@ -9,20 +9,56 @@ jobs:
         Linux_Node10X:
           OSVmImage: "ubuntu-16.04"
           TestType: "node"
+          NodeVersion: "10.x"
+          EVENTHUB_NAME: "hub_linux_node10"
         Windows_Node10X:
           OSVmImage: "vs2017-win2016"
           TestType: "node"
+          NodeVersion: "10.x"
+          EVENTHUB_NAME: "hub_windows_node10"
         Mac_Node10X:
           OSVmImage: "macOS-10.13"
           TestType: "node"
+          NodeVersion: "10.x"
+          EVENTHUB_NAME: "hub_mac_node10"
+        Linux_Node12X:
+          OSVmImage: "ubuntu-16.04"
+          TestType: "node"
+          NodeVersion: "12.x"
+          EVENTHUB_NAME: "hub_linux_node12"
+        Windows_Node12X:
+          OSVmImage: "vs2017-win2016"
+          TestType: "node"
+          NodeVersion: "12.x"
+          EVENTHUB_NAME: "hub_windows_node12"
+        Mac_Node12X:
+          OSVmImage: "macOS-10.13"
+          TestType: "node"
+          NodeVersion: "12.x"
+          EVENTHUB_NAME: "hub_mac_node12"
+        Linux_Node8X:
+          OSVmImage: "ubuntu-16.04"
+          TestType: "node"
+          NodeVersion: "8.x"
+          EVENTHUB_NAME: "hub_linux_node8"
+        Windows_Node8X:
+          OSVmImage: "vs2017-win2016"
+          TestType: "node"
+          NodeVersion: "8.x"
+          EVENTHUB_NAME: "hub_windows_node8"
+        Mac_Node8X:
+          OSVmImage: "macOS-10.13"
+          TestType: "node"
+          NodeVersion: "8.x"
+          EVENTHUB_NAME: "hub_mac_node8"
+
       EnvVars:
         AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
         AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
         AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-        ENDPOINT: $(js-event-hubs-test-endpoint)
-        EVENTHUB_CONNECTION_STRING: $(js-event-hubs-test-connection-string)
-        EVENTHUB_NAME: $(js-event-hubs-test-name)
         IOTHUB_CONNECTION_STRING: $(js-event-hubs-test-iothub-connection-string)
+        EVENTHUB_CONNECTION_STRING: $(js-event-hubs-test-connection-string)
+        ENDPOINT: $(js-event-hubs-test-endpoint)
 
       # Event Hubs tests do not support concurrent execution
```

Comment:
We should be careful about letting these all run in parallel as it might eat up our machine pool. However if you do keep this set to 0 then you should update the comment directly above as it no longer applies.

Reply:
How much max limit do we prefer to set for the pools to run in parallel?

Reply:
Looks like we have set this to 0 in other repo's like Python so lets go with that for now and we can adjust later if needed. Please at least remove the comment as it isn't correct any longer.

Reply:
Yes I will push the commit for that :)